protected/humhub/modules/friendship/widgets/FriendshipButton.php

```diff
@@ -20,10 +22,76 @@ class FriendshipButton extends \yii\base\Widget
 {
 
     /**
-     * @var User the target user 
+     * @var User the target user
      */
     public $user;
 
+    /**
+     * @var array Options buttons
+     */
+    public $options = [];
+
+    private function getDefaultOptions()
+    {
+        return [
+            'friends' => [
+                'title' => '<span class="glyphicon glyphicon-ok"></span>&nbsp;&nbsp;' . Yii::t('FriendshipModule.base', 'Friends'),
+                'attrs' => [
+                    'data-method' => 'POST',
+                    'data-confirm' => Yii::t('FriendshipModule.base', 'Would you like to end your friendship with {userName}?', ['{userName}' => '"' . $this->user->getDisplayName() . '"']),
```

Comment:
Html::encode?

Reply:
No need, because this array is used as 3rd param `$options` for the function `Html::a($text, $url = null, $options = [])`, then this param goes into `BaseHtml::renderTagAttributes($attributes)` where each value is properly encoded by the `Html::encode()` and other similar methods.

Reply:
@yurabakhtin Thanks!